### PR TITLE
[Mailer] E-mail teacher/student + admin upon completing signup form

### LINKED ISSUE #172: [Mailer] E-mail student upon finishing the sign up form

### LINKED ISSUE #189: [Email] Send email to teacher and admin when teacher submits sign up form

### PR DESCRIPTION
**Pull request by lilshim**
Fix #172 
Fix #189 

image
image
image

Testing Signup Procedures:
1. Fill out sign up form & submit 
2. Email should pop up 

Testing Contact Procedures:
1. Fill out contact form & submit 
2. Email should pop up

### CODE REVIEW

File: app/assets/javascripts/components/form/student_form.es6.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
 class StudentForm extends React.Component {
-  
+
```
Comment (lilshim) [inline app/assets/javascripts/components/form/student_form.es6.jsx]: :P this is why all our devs should have 
```
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true,
	"ui_font_size_small": true,
	"ui_separator": true
}
``` in their sublime settings~~~!!!!!!!!!!!! @sonia-y

File: app/models/student.rb [hunk 2]
```diff
@@ -55,42 +55,41 @@ class Student < ActiveRecord::Base
...
-  validates :household_number, presence: true
-  validates :disciplinary_action, :inclusion => { :in => [true, false] }
-  validates :criminal_charges, :inclusion => { :in => [true, false] }
-  validates :waiver_signature, presence: true
-  validates :waiver_date, presence: true
+  # validates :city, presence: true
+  # validates :first_name, presence: true
+  # validates :last_name, presence: true
+  # validates :email, presence: true, uniqueness: true
+  # validates :availability, presence: true
```
Comment (sonia-y) [inline app/models/student.rb]: uncomment this pls!!!

File: app/models/teacher.rb [hunk 3]
```diff
@@ -31,18 +31,28 @@ class Teacher < ActiveRecord::Base
...
-  validates :first_name, presence: true
-  validates :last_name, presence: true
-  validates :is_searching, :inclusion => { :in => [true, false] }
-  validates :availability, presence: true
+
+  # validates :email, presence: true, uniqueness: true
+  # validates :city, presence: true
+  # validates :first_name, presence: true
+  # validates :last_name, presence: true
+  # validates :is_searching, :inclusion => { :in => [true, false] }
```
Comment (sonia-y) [inline app/models/teacher.rb]: uncomment too

**Comment by sonia-y** [comment]
ty for figuring this out :) :) can you merge after you comment the validations

**Pull request merged**